actionlist/commons.js

```diff
@@ -1,18 +1,10 @@
-  return str;
+export function getProcessInstanceJSON() {
```

Comment:
I don't see a real benefit to having a function here. Can you change it to just `export const processInstanceJSON = '{"id":"12345","...":"..."}';` (same with the preference JSON).